packages/dht/src/dht/routing/RoutingSession.ts

```diff
@@ -13,29 +13,30 @@ import { EXISTING_CONNECTION_TIMEOUT } from '../contact/RpcRemote'
 import { getPreviousPeer } from './getPreviousPeer'
 import { DhtAddress, areEqualPeerDescriptors, getDhtAddressFromRaw, getNodeIdFromPeerDescriptor } from '../../identifiers'
 import { pull } from 'lodash'
+import { RoutingTableCache } from './RoutingTableCache'
 
 const logger = new Logger(module)
 
 const MAX_FAILED_HOPS = 2
-const CONTACT_LIST_MAX_SIZE = 10
+const ROUTING_TABLE_MAX_SIZE = 20
```

Comment:
Maybe could be more explicit:

```suggestion
const ROUTING_TABLE_MAX_CONTACT_COUNT = 20
```

Reply:
I prefer `ROUTING_TABLE_MAX_SIZE `